### PR TITLE
adding learning objectives to early episodes

### PR DESCRIPTION
**Pull request by ChristinaLK**
When prepping to teach this week, I wrote out my own goals / learning objectives and am adding them to the episodes.

### CODE REVIEW

File: _episodes/05-memory.md [hunk 1]
```diff
@@ -7,8 +7,11 @@ questions:
 - "What is cognitive load and how does it affect learning?"
 - "How can we design instruction to work with, rather than against, memory constraints?"
 objectives:
-- "Identify when cognitive load is likely to limit learning."
+- "Remember the quantitative limit of human memory." 
+- "Identify the number of concepts required in a lesson by drawing a concept map." 
 - "Use a concept map to plan a formative assessment strategy."
+- "Describe the three components of cognitive load theory and which should be emphasized or diminished."
```
Comment (karenword) [inline _episodes/05-memory.md:13]: While we do cover the three components specifically, we don't have an activity that asks learners to describe them. I find that most learner conversations focus on extraneous load and don't lean too heavily on the sub-terms (i.e. use "add to cognitive load" vs "create extraneous load").

File: _episodes/05-memory.md [hunk 2]
```diff
@@ -7,8 +7,11 @@ questions:
 - "What is cognitive load and how does it affect learning?"
 - "How can we design instruction to work with, rather than against, memory constraints?"
 objectives:
-- "Identify when cognitive load is likely to limit learning."
+- "Remember the quantitative limit of human memory." 
+- "Identify the number of concepts required in a lesson by drawing a concept map."
```
Comment (karenword) [inline _episodes/05-memory.md:11]: Maybe "Apply a concept map to identify concepts and relationships required in a lesson"? That more accurately places this as an advanced-Blooms activity (arguably too advanced).

File: _episodes/05-memory.md (hunk 1 above)
Comment (ChristinaLK) [reply _episodes/05-memory.md]: I changed "Describe" to "Name"

File: _episodes/02-practice-learning.md [hunk 3]
```diff
@@ -8,8 +8,11 @@ questions:
 - "Who is a typical Carpentries learner?"
 - "How can we help novices become competent practitioners?"
 objectives:
+- "Compare and contrast the three stages of skill acquisition."
+- "Identify a mental model and explain what it represents." 
 - "Explain why practice and feedback are important for learning."
-- "Use formative assessments to guide your teaching."
+- "Explain the purpose of formative assessments."
+- "Give an example of a formative assessment and when you would use it."
```
Comment (brownsarahm) [inline _episodes/02-practice-learning.md:15]: I *like* this objective, but I think that this episode functions more at the level of identifying formative assessments in past experiences, with some application specifically for identifying misconceptions with MCQ. Generating formative assessments generally happens in `15-lesson-study`.

File: _episodes/05-memory.md [hunk 4]
```diff
@@ -7,8 +7,11 @@ questions:
 - "What is cognitive load and how does it affect learning?"
 - "How can we design instruction to work with, rather than against, memory constraints?"
 objectives:
-- "Identify when cognitive load is likely to limit learning."
+- "Remember the quantitative limit of human memory." 
+- "Apply a concept map to identify concepts and relationships required in a lesson."
```
Comment (brownsarahm) [inline _episodes/05-memory.md:11]:
```suggestion
- "Apply concept mapping as a tool to identify concepts and relationships required in a lesson."
```

File: _episodes/02-practice-learning.md (hunk 3 above)
Comment (ChristinaLK) [reply _episodes/02-practice-learning.md:15]: I was building off of this exercise: Based on your previous educational experience (or even this training so far!) what types of formative assessments do you know about?

So I do think that participants should be able to give an example of a formative assessment, beyond MCQs. Maybe take off the "and when you would use it"?

Comment (brownsarahm) [reply _episodes/02-practice-learning.md:15]: That makes sense, thinking a little more, I think the issue is that on my first reading, "give an example" evoked the create level of bloom's but this episode I think covers more the understand level.  Maybe something like this?

```suggestion
- "Identify formative assessments in your teaching and learning experiences." 
```

Comment (ChristinaLK) [reply _episodes/02-practice-learning.md:15]: I'm not sure that quite captures what I'm going for which is if someone stopped you on the street, you could give them an example, but I do think it captures what happens in the lesson, so let's go with this for now.

**Comment by ChristinaLK** [comment]
Okay, I think this is ready for another review!

**Pull request merged**